### PR TITLE
OpenForBC proposal

### PR DESCRIPTION
**Pull request by leggerf**
Adding OpenForBC GSOC proposal

### CODE REVIEW

File: _gsocproposals/2021/proposal_openForBC.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+The successful student will develop and test a suite of ready-to-run benchmarks to measure the performances of various GPU partitioning options. 
+The student will perform the baseline performance measurements that are necessary to test the solutions developed within OpenForBC.
+
+## Requirements
+Bash, Python (C++ optional but appreciated), familiar with Linux environment (bonus skill: KVM, ESXi) and virtualization techniques
+
+## Mentors
+  * [Federica Legger](mailto:federica.legger@cern.ch)
+  * [Gabriele Gaetano Fronzé](mailto:gabriele.fronze@to.infn.it)
```
Comment (agheata) [inline _gsocproposals/2021/proposal_openForBC.md:41]: The main mentor should appear in **bold** here

File: gsoc/2021/mentors.md [hunk 2]
```diff
@@ -17,3 +17,5 @@ layout: plain
 * Olivier Mattelaer [olivier.mattelaer@uclouvain.be](mailto:olivier.mattelaer@uclouvain.be) UCLouvain
 * Stefan Roiser [stefan.roiser@cern.ch](mailto:stefan.roiser@cern.ch) CERN
 * Andrea Valassi [andrea.valassi@cern.ch](mailto:andrea.valassi@cern.ch) CERN
+* Federica Legger [federica.legger@cern.ch](mailto:federica.legger@cern.ch) INFN
+* Gabriele Gaetano Fronzè [gabriele.fronze@to.infn.it](mailto:gabriele.fronze@to.infn.it) INFN
```
Comment (agheata) [inline gsoc/2021/mentors.md:21]: Mentors have to be inserted in last name order in this list

**Review by agheata — CHANGES_REQUESTED** [review]

File: _gsocproposals/2021/proposal_openForBC.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+## Expected results
+
+The successful student will develop and test a suite of ready-to-run benchmarks to measure the performances of various GPU partitioning options. 
+The student will perform the baseline performance measurements that are necessary to test the solutions developed within OpenForBC.
+
+## Requirements
+Bash, Python (C++ optional but appreciated), familiar with Linux environment (bonus skill: KVM, ESXi) and virtualization techniques
+
+## Mentors
```
Comment (xvallspl) [inline _gsocproposals/2021/proposal_openForBC.md:39]: Main mentor should be in bold

**Review by xvallspl — CHANGES_REQUESTED** [review]

**Comment by leggerf** [comment]
the requested changes have been applied

File: _gsocproposals/2021/proposal_openForBC.md (hunk 3 above)
Comment (leggerf) [reply _gsocproposals/2021/proposal_openForBC.md:39]: ok, changed

File: gsoc/2021/mentors.md (hunk 2 above)
Comment (leggerf) [reply gsoc/2021/mentors.md]: fixed

**Review by agheata — APPROVED** [review]

**Comment by agheata** [comment]
Looks good to me, I'll fix the conflict in mentors.md file once @xvallspl approves

**Review by xvallspl — APPROVED** [review]

**Comment by xvallspl** [comment]
Done!

**Comment by agheata** [comment]
Well, it looks like I cannot fix the mentors.md conflict online, to have a clean history, can you do it @leggerf ?

**Comment by leggerf** [comment]
can you try now?

**Comment by agheata** [comment]
Still conflicts, you need to rebase on the master or apply the instructions proposed above to fix the conflict in this file

**Comment by leggerf** [comment]
I don't see any more conflicts, can you try again the merge?

**Comment by agheata** [comment]
Thanks!

**Pull request merged**